django/db/backends/oracle/utils.py

```diff
@@ -40,3 +40,26 @@ def from_datetime(cls, dt):
             dt.year, dt.month, dt.day,
             dt.hour, dt.minute, dt.second, dt.microsecond,
         )
+
+
+class BulkInsertMapper:
```

Comment:
I have a small concern that it's not easy for custom fields that use other types to add their own types to the mapper, but this problem exists elsewhere so I guess it's okay to use this pattern for now.